tests/test_elastic.py

```diff
@@ -270,6 +271,14 @@ def test_newPoint():
     assert point.last_modified_by == "some sub"
     assert point.created_by == "some sub"
 
+def test_newPoint_valueError():
+    with ShouldRaise(ValueError ('point type is not on point type list')):
```

Comment:
There is no need to use new library, we can use `pytest.raise`, example here https://github.com/wiating-app/wiating_backend/blob/master/tests/test_auth.py#L17